lib/rollbar/middleware/rails/rollbar.rb

```diff
@@ -21,6 +21,7 @@ def call(env)
 
           Rollbar.scoped(scope) do
             begin
+              Rollbar.notifier.enable_locals
```

Comment:
What's the reason behind enabling and disabling here? Is it that you want this to be the default behavior only for Rails?

Reply:
These will be added to the other middleware as well. The intent is to only have the TracePoint hooks turned on when needed, to minimize the perf impact.

Reply:
Oh yeah you mentioned that. Awesome